src/js/edu-benefits/5490/config/form.js

```diff
@@ -34,6 +47,10 @@ const formConfig = {
   confirmation: ConfirmationPage,
   title: 'Update your Education Benefits',
   subTitle: 'Form 22-5490',
+  defaultDefinitions: {
+    educationType: educationType.schema,
```

Comment:
@cvalarida I'm going to leave this, pending how the change to set all the $ref fields goes (https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1578)